app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -2629,6 +2630,23 @@ class PluginApi {
       .lookup("service:admin-custom-user-fields")
       .addProperty(userFieldProperty);
   }
+
+  /* Generates a visual feedback indicator for a specific post action.
+   *
+   * For example: when you click the "chain" on a post you see a visual indicator it
+   * was copied, with a checkmark and text that hovers above it saying it was copied.
+   *
+   * @param {Object} opts - object with the following properties:
+   *
+   * @param {Integer} opts.postId - the post the action belongs on
+   * @param {String} opts.actionClass - css class of the post action menu button
+   * @param {String} opts.messageKey - I18n key for message to be displayed in the feedback indicator
+   * @param {function} opts.actionCallback - callback to be made if action is to be taken, can be a promise
+   * @param {function} opts.errorCallback - optional callback to be called if the post action actionCallback fails.
```

Comment:
```suggestion
   * @param {function} [opts.errorCallback] - callback to be called if the post action actionCallback fails.
```

In JSDoc, optional params are indicated by wrapping the param name in `[]`